sdk/entra/functions-authentication-events/review/functions-authentication-events.api.md

```diff
@@ -0,0 +1,144 @@
+## API Report File for "@azure/functions-authentication-events"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @public (undocumented)
```

Comment:
Please add docs to each interface or properties marked with (undocumented)

Reply:
Done